content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst

```diff
@@ -39,7 +39,7 @@ setting, :guilabel:`Save` the changes.
    Activating the :guilabel:`Multi-Step Routes` setting will also activate the :guilabel:`Storage
    Locations` feature.
 
-.. image:: receipts_delivery_two_steps/multi-step-routes.png
+.. image:: receipts_delivery_two_steps/receipts-delivery-two-steps-settings.png
```

Comment:
Not against it...just don't see the change here other than adding 10kb.

Reply:
The old screenshot was in a 15.0 db, and was replaced with a screenshot from a 17.2 database. It does look similar, but there are very small differences - the text is slightly thinner and less bold